### PR TITLE
Updated requirements.in

### PR DESCRIPTION
**Pull request by dingus9**
fixes error where molecule-openstack requires openstacksdk

This error only appears when using molecule-openstack as the driver

### CODE REVIEW

**Comment by tadeboro** [comment]
@dingus9 You will also need to update the requirements.txt because that file is tracked in repo. Use pip-compile to generate the file.

**Comment by dingus9** [comment]
> @dingus9 You will also need to update the requirements.txt because that file is tracked in repo. Use pip-compile to generate the file.

I assumed that would be part of the the pipeline ;)

**Comment by dingus9** [comment]
Hey not exactly sure why it says git is in a dirty status... can you point me in the right direction

File: requirements.txt [hunk 1]
```diff
@@ -2,11 +2,12 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --allow-unsafe --no-annotate --output-file=requirements.txt requirements.in
+#    pip-compile --allow-unsafe --no-annotate --output-file=requirements2.txt requirements.in
```
Comment (tadeboro) [inline requirements.txt:5]: This line should not be changed because instructions did not change. This is what is triggering the CI.

**Review by tadeboro — CHANGES_REQUESTED** [review]

**Comment by ssbarnea** [comment]
@dingus9 Rebase and run tox locally before uploading. If tox is passing locally it will also pass on CI.

**Comment by dingus9** [comment]
> @dingus9 Rebase and run tox locally before uploading. If tox is passing locally it will also pass on CI.

why does tox dirty my commit? WTF

**Comment by ssbarnea** [comment]
That is because you did not run it locally, the linter job does regenerate the requirements and finishes with dirty-status failure if the regenerated file is different than the one in the commit. That assures we do not merge manually modified files.

**Comment by dingus9** [comment]
ugh I'm running tox locally tox is passing, did a merge to origin main... IDK what you guys are doing, why isn't the requirements.txt just an output of a pipeline stage? I'm just going to close it's wasting everyones time.

**Comment by ssbarnea** [comment]
@dingus9 Yeah, is not your fault. I am working on it, it needs some extra changes to the pipeline because the result of the pip-compile depends on python/platform being used, this being the root cause.  I hope i get it to work by pinning python version.

**Pull request closed without merging**